moskito-core/src/main/java/net/anotheria/moskito/core/predefined/OSStats.java

```diff
@@ -87,20 +87,18 @@ public OSStats(String aName,  Interval[] selectedIntervals){
 
 	@Override
 	public String toStatsString(String intervalName, TimeUnit unit) {
```

Comment:
I find this version, with all + signs, harder to maintain, why should we change to it? And honestly I don't believe its a microsecond faster...